src/js/Controllers/TTSController.js

```diff
@@ -33,7 +36,11 @@ class TTSController {
         this.audioPlayer.play();
     }
 
-    speak(text) {;
+    speak(text) {
+        // Dont allow empty strings
+        if(!text) {
+            this.playNext();
+        }
```

Comment:
```suggestion
```

Core already prevents empty strings

Reply:
@iCollin please find the changes [here](https://github.com/LuxoftSDL/generic_hmi/pull/39/files).